src/battle_util.c

```diff
@@ -10828,6 +10828,7 @@ bool32 ShouldGetStatBadgeBoost(u16 badgeFlag, u32 battler)
 
 u8 GetBattleMoveCategory(u32 moveId)
 {
+    u8 moveType;
```

Comment:
This should be `moveCategory`, right?

Reply:
No, it's move type since the pre-PSS way of deciding was by type, and this is the value the type is saved to, and it had to be here to make `agbcc` stop complaining.

Reply:
Makes sense. Could you please force push with a clean commit? I messed up a little when I wanted to merge after removing the brackets